plugins/widgetselection/plugin.js

```diff
@@ -22,10 +22,9 @@
 				editor.on( 'contentDom', function( evt ) {
 
 					var editor = evt.editor,
-						doc = editor.document,
 						editable = editor.editable();
 
-					editable.attachListener( doc, 'keydown', function( evt ) {
+					editable.attachListener( editable, 'keydown', function( evt ) {
```

Comment:
I would be nice to provide some simple explanation comment with the issue ticket reference for this change.

Reply:
TBH I don't think it is needed there, it's fairly logical that `ctrl/cmd+a` should be catched from within the editable. It's actually shame that we didn't catch this earlier, because in fact it didn't make sense in case of inline editors. 🙂